atlassian_jwt_auth/contrib/tests/django/test_django.py

```diff
@@ -19,6 +19,124 @@ def create_token(issuer, audience, key_id, private_key):
     return signer.generate_jwt(audience)
 
 
+@modify_settings(MIDDLEWARE={
+    'prepend': 'atlassian_jwt_auth.contrib.django.middleware.ASAPMiddleware',
+})
+class TestAsapMiddleware(RS256KeyTestMixin, SimpleTestCase):
+
+    @classmethod
+    def setUpClass(cls):
```

Comment:
Can we avoid duplication of code here by sharing the similar aspects of `TestAsapDecorator` and `TestAsapMiddleware` ? (e.g. `setUpClass`, `tearDownClass`, `setUp`)